src/olympia/amo/management/commands/compress_assets.py

```diff
@@ -0,0 +1,302 @@
+import hashlib
+import os
+import re
+import shutil
+import time
+import urllib2
+import uuid
+
+from subprocess import PIPE, call, check_output
+
+from django.conf import settings
+from django.core.management.base import BaseCommand, CommandError
+
+from olympia.lib.jingo_minify_helpers import get_media_root, get_path
+
+
+def path(*args):
+    return os.path.join(get_media_root(), *args)
+
+
+class Command(BaseCommand):
+    help = ('Compresses css and js assets defined in settings.MINIFY_BUNDLES')
+    requires_model_validation = False
+    do_update_only = False
+
+    checked_hash = {}
+    bundle_hashes = {}
+
+    missing_files = 0
+    minify_skipped = 0
+    cmd_errors = False
+    ext_media_path = os.path.join(get_media_root(), 'external')
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--use-uuid', action='store_true', dest='use_uuid',
+            help='Use a uuid as the build id instead of git.')
+        parser.add_argument(
+            '-u', '--update-only', action='store_true', dest='do_update_only',
+            help='Updates the hash only')
+        parser.add_argument(
+            '-t', '--add-timestamp', action='store_true', dest='add_timestamp',
+            help='Add timestamp to hash')
+
+    def generate_build_id(self, use_uuid):
+        if use_uuid:
+            return uuid.uuid4().hex[:8]
+        else:
+            root = getattr(settings, 'JINGO_MINIFY_ASSETS_GIT_ROOT', '.')
+            git_bin = getattr(settings, 'GIT_BIN', 'git')
+            return check_output(
+                [git_bin, '-C', root, 'rev-parse', '--short', 'HEAD']).strip()
```

Comment:
both places we call this command we do `compress_assets --use-uuid` so may as well drop the git support and simplify this code.